internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+# Script Receiver
+
+<!-- status autogenerated section -->
+| Status        |                        |
+| ------------- |------------------------|
+| Stability     | [in-development]: logs |
+| Distributions | [contrib]              |
+
+[in-development]: https://github.com/open-telemetry/opentelemetry-collector#in-development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+## Overview
+Script receiver is a component that performs log collection equivalent to what the UF does when Unix and Linux Technical Add-on is installed.
+
+
+| Script name           | Description                                                                                                                                                                                                                  |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| bandwidth.sh	         | Network statistics via the shell commands dlstat, netstat, and sar                                                                                                                                                           |
```

Comment:
Are the `.sh` extensions necessary? If this is from existing config conventions I can see that being the case, but they could be redundant.